lib/system/allocator.cpp

```diff
@@ -8,7 +8,7 @@
 #include "common/errcode.h"
 
 #if defined(HAVE_MMAP) && defined(__x86_64__) || defined(__aarch64__) ||       \
-    defined(__arm__)
+    defined(__arm__) || defined(__riscv)
```

Comment:
Please make sure the `riscv` target supports MMAP.

Reply:
Yes, there is the following description in `sys/mman.h`.
```
#ifndef __USE_FILE_OFFSET64
extern void *mmap (void *__addr, size_t __len, int __prot,
		   int __flags, int __fd, __off_t __offset) __THROW;
#else
# ifdef __REDIRECT_NTH
extern void * __REDIRECT_NTH (mmap,
			      (void *__addr, size_t __len, int __prot,
			       int __flags, int __fd, __off64_t __offset),
			      mmap64);
# else
#  define mmap mmap64
# endif
#endif
```

Reply:
Okay.